styles/style_skills.css

```diff
@@ -37,3 +38,25 @@
   align-self: center;
   margin: 0 0 40px 0;
 }
+
+@media (max-width: 1000px) {
```

Comment:
dodaj margin do diza z przyciskami żeby na siebie nie nachodziły :)

Reply:
W mojej sekcji nie mam przycisków, chyba chodziło Ci, Klara, o te z sekcji title. To bodajże Szymon będzie ogarniał, więc ja zmerguję swoje i zobaczymy co z tego wyjdzie.